models/rollup_economics/ethereum/rollup_economics_ethereum_l1_fees.sql

```diff
@@ -0,0 +1,136 @@
+{{ config(
+    alias = alias('l1_calldata_fees'),
+    tags = ['dunesql'],
+    partition_by = ['block_time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "rollup_economics",
+                                    \'["niftytable"]\') }}'
+)}}
+
+WITH tx_batch_appends AS (
+    SELECT
+    'Arbitrum' AS name,
+    t.block_time,
+    t.block_number,
+    t.hash,
+    (cast(gas_used as double) * (cast(gas_price as double) / 1e18)) AS gas_spent,
+    length(t.data) AS data_length
+    FROM
+    (
+      SELECT
+      evt_tx_hash AS tx_hash,
+      evt_block_time AS block_time,
+      evt_block_number AS block_number
+      FROM source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') o
```

Comment:
This should be in brackets I believe throughout.

like :
{{ source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') }}